contracts/mock/liquity/MockLiquityStrategyV3.sol

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: MIT
+pragma solidity =0.8.10;
+
+import {LiquityStrategy} from "../../strategy/liquity/LiquityStrategy.sol";
+
+contract MockLiquityStrategyV3 is LiquityStrategy {
```

Comment:
Why do we need 3 versions of a mock liquity?

Reply:
i think there are only two, the first one is called V2, not sure why. I added V3 because putting these changes into V2 broke the tests

Reply:
@brkomir can you have a look? I'm still confused as to why we need so many mock versions

Reply:
the V2 version was added for the purpose of writing proxy tests, nothing else. I think the V3 brought confusion here, maybe just remove it. Also, we could make things work with only one mock, I suppose the reason tests were failing is that in V2 the `harvest` function is overridden to revert

Reply:
yeah that's exactly why I added V3

How can we combine the logic? Not sure which condition the revert could rely on in this case to be honest

Reply:
you can make the `harvest` not revert and change the proxy test for that since it was just an arbitrary logic

Reply:
@0xrin1 can you address this thread?